docs/configuring-metabase/caching.md

```diff
@@ -7,50 +7,125 @@ redirect_from:
 
 # Caching query results
 
-Metabase now gives you the ability to automatically cache the results of queries that take a long time to run.
+If your results don't change frequently, you may want to cache your results, that is: store your results in Metabase so that the next time you visit the question, Metabase can retrieve the stored results rather than query the database again. 
 
-## Enabling caching
+For example, if your data only updates once a day, there's no point in querying the database more than once a day, as they data won't have changed. Returning cached results can be significantly faster, as the database won't have to redo the work to answer your query.
 
-To start caching your queries, head to the Settings section of the Admin Panel, and click on the `Caching` tab at the bottom of the side navigation. Then turn the caching toggle to `Enabled`.
+Metabase gives you the ability to automatically cache query results. On [paid plans](https://www.metabase.com/pricing/), you can also control caching for specific:
 
-![Caching](images/caching.png)
+- [Databases](#caching-per-database)
+- [Questions](#caching-per-question)
+- [Dashboards](#caching-per-dashboard)
 
-End-users will see a timestamp on cached questions in the top right of the question detail page showing the time when that question was last updated (i.e., the time when the current result was cached). Clicking on the `Refresh` button on a question page will manually rerun the query and override the cached result with the new result.
+You can also [cache models](../data-modeling/models.md#caching-individual-models).
```

Comment:
Why is model caching excluded/separate from the list above?

Not for this PR, just noting for myself: how does caching work with actions? Will enabling model actions disable caching for models with actions? (I can follow up with a PR to answer this, if necessary).